pkg/cmd/cli/cmd/rsync/rsync.go

```diff
@@ -111,6 +117,7 @@ func NewCmdRsync(name, parent string, f *clientcmd.Factory, out, errOut io.Write
 	cmd.Flags().StringVar(&o.RsyncInclude, "include", "", "rsync - include files matching specified pattern")
 	cmd.Flags().BoolVar(&o.RsyncProgress, "progress", false, "rsync - show progress during transfer")
 	cmd.Flags().BoolVar(&o.RsyncNoPerms, "no-perms", false, "rsync - do not transfer permissions")
+	cmd.Flags().BoolVarP(&o.Watch, "watch", "w", false, "Watch directory for changes and resync automatically")
```

Comment:
if by "consistent" with get you mean that get has a --watch flag, then yes.  Whether what get --watch does is similar enough to rsync --watch that they should be using the same argument name is debatable.  (get is reading something and display you the result whenever it changes.  rsync is monitoring the filesystem for changes and sending them to the container whenever it changes).  So...similarish, not identical.

would we ever want the equivalent of get --watch functionality in rsync?  I have a hard time envisioning that we would.

overall watch feels like the right name here i think.